WordPress/WordPressTest/PostTests.swift

```diff
@@ -376,6 +376,54 @@ class PostTests: XCTestCase {
         XCTAssertFalse(revision.hasLocalChanges())
     }
 
+    func testThatVersionConflictWorksShouldNotConflict() {
+        let original = newTestPost()
+        XCTAssertFalse(original.hasVersionConflict(), "Post should return false for `hasVersionConflict`")
+    }
+
+    // Local changes alone should not trigger `hasVersionConflict`
+    func testThatVersionConflictWorksShouldNotConflictWithOnlyTagChange() {
+        let original = newTestPost()
+        let local = original
```

Comment:
I'm not sure I follow this.

`local` and `original` are the same entity (they don't have an `original`, for example). When you create a local change based on a post you'll have two entities: the new `Post` entity and the `original` one.